cheatsheets/JSON_Web_Token_Cheat_Sheet_for_Java.md

```diff
@@ -305,19 +305,19 @@ public class TokenRevoker {
  }
 ```
 
-## Token information disclosure
+## Token Information Disclosure
 
 ### Symptom
 
-This attack occur when a attacker access to a token (or a set of tokens) and extract information stored into it (JWT token information are base64 encoded at the basis) in order to obtains information about the system. Information can be for example the security roles, login format...
+This attack occurs when an attacker has access to a token (or a set of tokens) and extracts information stored in it (JWT token information are base64 encoded at the basis) in order to obtain information about the system. Information can be for example the security roles, login format...
```

Comment:
I'm not sure what `at the basis` is meant to mean here. I think the brackets should be something like:

`(the contents of JWT tokens is base64 encoded, but is not encrypted by default)`